@hesburgh-wse/gatsby-theme-marble/src/components/Layout/PageWrapper/BrandingHeader/index.js

```diff
@@ -1,93 +1,9 @@
+// eslint-disable-next-line
```

Comment:
I'd prefer an explicit declaration to disable the rule `no-unused-vars`. That way if somebody fat-fingers a different eslint violation it will still be caught.